docs/flow-visibility.md

```diff
@@ -0,0 +1,85 @@
+|---------------------------|---------------|----------|-------------|
+| sourcePodNamespace        | 55829         | 100      | string      |
+| sourcePodName             | 55829         | 101      | string      |
+| destinationPodNamespace   | 55829         | 102      | string      |
+| destinationPodName        | 55829         | 103      | string      |
+| sourceNodeName            | 55829         | 104      | string      |
+| destinationNodeName       | 55829         | 105      | string      |
+| destinationClusterIP      | 55829         | 106      | ipv4Address |
+| destinationServicePort    | 55829         | 107      | unsigned16  |
+| destinationServicePortName| 55829         | 108      | string      |
+
+[Elasticsearch](https://www.elastic.co/elasticsearch/), as a RESTful search
+engine, supports storing, searching and indexing records received. 
+[Kibana](https://www.elastic.co/kibana/) is mainly for data visualization and
+exploration.
+
+## Deployment Steps
+To create all the necessary resources in the `antrea-flow-collector` namespace
+and get
+everything up-and-running, run:
+```shell
+kubectl create namespace antrea-flow-collector
+kubectl create configmap logstash-configmap -n antrea-flow-collector --from-file=build/yamls/flow-collector/logstash/
+kubectl apply -f build/yamls/flow-collector/flow-collector.yml -n antrea-flow-collector
+```
+Kibana dashboard is exposed as a Nodeport, which can be accessed via
+`http://[NodeIP]: 30007`
+
+To import the pre-built and recommended dashboard into Kibana, go to
+**Management -> Saved Objects** and
+import `build/yamls/flow/kibana.ndjson`
+
+
+## Pre-built Dashboards
+The following dashboards are pre-built and recommended for Antrea flow
+visualization.
+
+### Overview
+<img src="http://downloads.antrea.io/static/flow-visualization-overview.png" width="900" alt="Flow
+Visualization Overview Dashboard"> 
+
+### Flows
+#### Pod-to-pod Traffic
+<img src="http://downloads.antrea.io/static/flow-visualization-flow-1.png" width="900" alt="Flow
+Visualization Flows Dashboard"> 
+<img src="http://downloads.antrea.io/static/flow-visualization-flow-2.png" width="900" alt="Flow
+Visualization Flow Dashboard"> 
+
+#### Pod-to-service Traffic
+<img src="http://downloads.antrea.io/static/flow-visualization-flow-3.png" width="900" alt="Flow
+Visualization Flows Dashboard"> 
+<img src="http://downloads.antrea.io/static/flow-visualization-flow-4.png" width="900" alt="Flow
+Visualization Flow Dashboard"> 
+
+### Flow Records 
+<img src="http://downloads.antrea.io/static/flow-visualization-flow-record.png" width="900" alt="Flow
+Visualization Flow Record Dashboard">
+
+### Node Throughput
+<img src="http://downloads.antrea.io/static/flow-visualization-node-1.png" width="900" alt="Flow
+Visualization Node Throughput Dashboard">
+<img src="http://downloads.antrea.io/static/flow-visualization-node-2.png" width="900" alt="Flow
+Visualization Node Throughput Dashboard">
```

Comment:
when I look at the rendered markdown file, this part is really packed with images. Having these images is great, but do you think you could add a line or 2 of test for every section to break the flow of images and explain what is being shown?

Additionally, I would really recommend using the full AWS S3 link even though it is longer (which is not an issue IMO since the link is not used by users), in order to load images over HTTPs.

Reply:
Agrees. Added some elaboration and changed the image link